package.json

```diff
@@ -36,32 +36,34 @@
     "ng-packagr"
   ],
   "peerDependencies": {
-    "@angular/common": "^6.0.0",
-    "@angular/core": "^6.0.0",
-    "@angular/forms": "^6.0.0",
-    "@acpaas-ui/ngx-components": "^3.0.0"
+    "@angular/common": "^8.2.0",
+    "@angular/core": "^8.2.0",
+    "@angular/forms": "^8.2.0",
+    "@acpaas-ui/ngx-forms": "^4.2.2",
+    "@acpaas-ui/ngx-utils": "^4.2.1"
   },
   "dependencies": {},
   "devDependencies": {
     "@a-ui/core": "~4.1.0",
     "@a-ui/flexboxgrid": "~1.0.1",
-    "@acpaas-ui/ngx-components": "^3.0.0",
-    "@angular-devkit/build-angular": "~0.8.0",
-    "@angular/animations": "6.1.9",
-    "@angular/cli": "^6.2.3",
-    "@angular/common": "6.1.9",
-    "@angular/compiler": "6.1.9",
-    "@angular/compiler-cli": "6.1.9",
-    "@angular/core": "6.1.9",
-    "@angular/forms": "6.1.9",
-    "@angular/language-service": "6.1.9",
-    "@angular/platform-browser": "6.1.9",
-    "@angular/platform-browser-dynamic": "6.1.9",
-    "@angular/router": "6.1.9",
+    "@acpaas-ui/ngx-forms": "^4.2.2",
```

Comment:
Idem hier: `^4.0.0`

Reply:
Niet mogelijk omdat er in versie 4.2.2 een bugfix zit die belangrijk is voor de werking van de contact picker component.